routers/match.py

```
@@ -0,0 +1,16 @@
+from fastapi import APIRouter, Depends, Query, HTTPException
```

Ruff (0.11.9)

`fastapi.HTTPException` imported but unused

Remove unused import: `fastapi.HTTPException`

(F401)